src/sentry/models/group.py

```diff
@@ -158,24 +158,13 @@ def from_event_id(self, project, event_id):
         Resolves the 32 character event_id string into
         a Group for which it is found.
         """
-        from sentry.models import EventMapping, Event
+        from sentry.models import SnubaEvent
         group_id = None
 
-        # Look up event_id in both Event and EventMapping,
-        # and bail when it matches one of them, prioritizing
-        # Event since it contains more history.
-        for model in Event, EventMapping:
```

Comment:
Discussed in https://github.com/getsentry/sentry/pull/13905/files/3d3aa8be9807b50b6548c0b1b5a6f5b1027b5f4e#r301352764

removing EventMapping dependency should be safe.